src/config/artists.rs

```diff
@@ -28,8 +41,26 @@ pub enum AlbumSortMode {
     Date,
 }
 
+fn default_album_date_tags() -> Vec<String> {
+    vec!["date".to_string()]
+}
+
+impl Default for Artists {
+    fn default() -> Self {
+        Self {
+            album_display_mode: AlbumDisplayMode::default(),
+            album_sort_by: AlbumSortMode::default(),
+            album_date_tags: default_album_date_tags(),
+        }
```

Comment:
```suggestion
        ArtistsFile::default().into()
```

Better to simply proxy the other struct's default impl to ensure that they have same defaults.